src/main/java/com/thealgorithms/dynamicprogramming/Fibonacci.java

```diff
@@ -80,15 +80,6 @@ public static int fibBotUp(int n) {
      * @author Shoaib Rayeen (https://github.com/shoaibrayeen)
```

Comment:
There are comments about the method that was implemented as an optimized Fibonacci.